test/test_log.py

```diff
@@ -168,25 +168,25 @@ def test_plugin_path_long_1(self):
         DebugOpt.PLUGIN_FULLPATH.enabled = True
         record = FakeRecord(name=None, pathname='/user/picard/plugins/plugin.zip')
         self.assertTrue(name_filter(record))
-        self.assertEqual(record.name, 'user/picard/plugins/plugin')
+        self.assertEqual(record.name, '/user/picard/plugins/plugin')
 
     def test_plugin_path_long_2(self):
         DebugOpt.PLUGIN_FULLPATH.enabled = True
-        record = FakeRecord(name=None, pathname='/user/picard/plugins/plugin.zip/plugin')
+        record = FakeRecord(name=None, pathname='/user/picard/plugins/plugin.zip/xxx')
         self.assertTrue(name_filter(record))
-        self.assertEqual(record.name, 'user/picard/plugins/plugin.zip/plugin')
+        self.assertEqual(record.name, '/user/picard/plugins/plugin.zip/xxx')
 
     def test_plugin_path_short_1(self):
         DebugOpt.PLUGIN_FULLPATH.enabled = False
         record = FakeRecord(name=None, pathname='/user/picard/plugins/plugin.zip')
         self.assertTrue(name_filter(record))
-        self.assertEqual(record.name, 'plugins/plugin')
+        self.assertEqual(record.name, 'plugin')
 
     def test_plugin_path_short_2(self):
         DebugOpt.PLUGIN_FULLPATH.enabled = False
-        record = FakeRecord(name=None, pathname='/user/picard/plugins/plugin.zip/plugin')
+        record = FakeRecord(name=None, pathname='/user/picard/plugins/plugin.zip/xxx')
         self.assertTrue(name_filter(record))
-        self.assertEqual(record.name, 'plugins/plugin')
+        self.assertEqual(record.name, 'xxx')
```

Comment:
In this case it looks weird to lose the plugin name (from `plugin.zip`), I guess having `plugin/xxx` as result would make much more sense.

Reply:
What do you think of prefacing all shortened plugin paths with "Plugin:" to clearly indicate that's what they are?  For example, "/home/user/picard/plugins/xxx.zip/yyy" would be logged as "Plugin: yyy" and "/home/user/picard/plugins/xxx.zip" would be logged as "Plugin: xxx" (assuming that `USER_PLUGIN_DIR` was "/home/user/picard/plugins").

Otherwise, I can just put it back to starting with "plugins/".  I'm okay either way.  Just let me know what you prefer.

Reply:
Hmmm, I now get why you added `plugins/` at first. And I guess that's the best option.

Reply:
Actually, in looking at the plugins, stripping out the zip file from the path might not be such a good idea because in some cases the file inside may be `__init__.py`.  Perhaps we should leave the zip file name in the path to help ensure that the plugin is properly identified.  That way, some examples might be:

* `Plugin: abbreviate_artistsort.zip/abbreviate_artistsort`
* `Plugin: add_to_collection.zip/__init__`
* `Plugin: extra/path/plugin.zip/__init__`

Or we could stop at the zip file and strip the ".zip", but that wouldn't show which file it was within the plugin (if there were multiple modules).  For example, the three above would be:

* `Plugin: abbreviate_artistsort`
* `Plugin: add_to_collection`
* `Plugin: extra/path/plugin`

Considering that we have the option to show the full path if needed, I'm actually tempted to do it this way.  What do you think?

Reply:
I'll put it back to just including the "plugin/" for now and we can change later if we decide we want to go a different route.

Reply:
Another option, and I think this might actually make the most sense, is to not add the "plugins" back in, but also not remove the zip file from the path.  This keeps it as a shortened (relative) path, and the presense of a zip file in the path clearly indicates that this is a plugin.  Some examples might be:

* `abbreviate_artistsort.zip/abbreviate_artistsort`
* `add_to_collection.zip/__init__`
* `some_plugin.zip/module_path/some_module`
* `extra/path/plugin.zip/__init__`